File: ex17.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char const *argv[])
{
    float  lado1, lado2, lado3;
    printf("digite o lado1 do triangulo:");
    scanf("%f", &lado1);

    printf("digite o lado2 do triangulo:");
    scanf("%f", &lado2);

    printf("digite o lado3 do triangulo:");
    scanf("%f", &lado3);


    if ((lado1*lado1)+(lado2*lado2)== lado3*lado3 || (lado1*lado1)+(lado3*lado3)== lado2*lado2 || (lado2*lado2)+(lado3*lado3)== lado1*lado1) {
     printf("triangulo retangulo:");
    }
   
    else{
    printf("nao e triangulo retangulo") ;    
    }

   


    return 0;
}

File: python/ex17.py
lado1 = float(input("entre com o valor 1:"))
lado2 = float(input("entre com o valor 2:"))
lado3 = float(input("entre com o valor 3:"))



if ((lado1*lado1)+(lado2*lado2)== lado3*lado3 or (lado1*lado1)+(lado3*lado3)== lado2*lado2 or (lado2*lado2)+(lado3*lado3)== lado1*lado1):
    print("é triangulo retangulo")
else:
    print("nao e trianhulo retangulo")    